app/models/case_log.rb

```diff
@@ -460,10 +464,69 @@ def self.label_from_value(value)
   end
 
   def self.csv_attributes(user)
```

Comment:
Wonder if it's worth splitting out the CSV specific stuff into it's own module or service class or something... this isn't really a suggestion just a pondering...

Reply:
Was thinking the same thing. I would've liked to chain all of the new commands as well, but unsure how. Very keen to refactor this to something nicer